### PR TITLE
Feat/click listeners

### PR DESCRIPTION
**Pull request by nattsnb**

### CODE REVIEW

File: Calculator.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        row.append(buttonContainer);
+        new DigitButton(
+            buttonsArray[i],
+            this,
+            buttonContainer,
+        );
+      }
+      else {
+        const buttonContainer = document.createElement("div");
+        row.append(buttonContainer);
```
Comment (Solnick) [inline Calculator.js]:
```suggestion
    for (let i = 0; i < buttonsArray.length; i++) {
      const buttonContainer = document.createElement("div");
      row.append(buttonContainer);
      if (Number(buttonsArray[i])) {
        new DigitButton(
            buttonsArray[i],
            this,
            buttonContainer,
        );
      }
      else {
```

Comment (Solnick) [reply Calculator.js]: duplicate code which can be elevated outside of the if statement

File: Input.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+export class Input {
+  constructor(inputContainer) {
+    this.inputContainer = inputContainer;
+    this.num1 = null;
+    this.num2 = null;
```
Comment (Solnick) [inline Input.js]: `num1` and `num2` are poor names for object properties. I have no idea what it represents.

File: Input.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    this.eqSign = null;
+  }
+  digitButtonPressed = (digit) => {
+    this.inputContainer.innerHTML = this.inputContainer.innerHTML + digit;
+  };
+  actionButtonPressed = (eq) => {
+    if (eq === "=") {
```
Comment (Solnick) [inline Input.js:40]: This function is hard to follow. Can you describe why we need all of this if statements? Then, from this point we can think of improving the readability.

Also get rid of the console.logs

File: Calculator.js [hunk 4]
```diff
@@ -31,9 +31,9 @@ export class Calculator {
     const row = document.createElement("div");
     row.classList.add(`row`);
     for (let i = 0; i < buttonsArray.length; i++) {
+      const buttonContainer = document.createElement("div");
```
Comment (Solnick) [inline Calculator.js:34]: The code djplicate is still there, but now it is executed twice

**Review by Solnick — CHANGES_REQUESTED** [review]
What about rest of the comments?

File: Input.js (hunk 3 above)
Comment (nattsnb) [reply Input.js:40]: yes, I think the statements are needed to fool prove the application. they prevent the app from bugging. deleted 2 of them from first statement, and added one more later, as I've found another bug trying to check if all are needed

**Review by Solnick — CHANGES_REQUESTED** [review]
please change code according to comments
